ui/pages/destinations/index.js

```diff
@@ -119,12 +119,12 @@ export default function Destinations () {
       </Head>
       {!loading && (
         <div className='flex-1 flex h-full'>
-          <main className='flex-1 flex flex-col space-y-4'>
+          <div className='flex-1 flex flex-col space-y-4'>
             <PageHeader header='Clusters' buttonHref={admin && '/destinations/add'} buttonLabel='Cluster' />
             {error?.status
               ? <div className='my-20 text-center font-light text-gray-300 text-sm'>{error?.info?.message}</div>
               : (
-                <div>
+                <div className='px-6 flex-1 min-h-0 overflow-y-scroll px-6'>
```

Comment:
small suggestion - i think we should move this into the `table.js` component.

Reply:
I like this suggestion to avoid code duplication, but since this also effects the `EmptyTable` component below and needs to be a parent for the table element we may want to keep it here.

That said, this is a great suggestion since we should consider pulling out this table logic into it's own component – there's quite a bit of repeated code between `destinations.js` `users.js` and `providers.js`